examples/trading/frontend/src/mutations/demo.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CONSTANTS } from "@/constants";
+import { useTransactionExecution } from "@/hooks/useTransactionExecution";
+import { useCurrentAccount } from "@mysten/dapp-kit";
+import { TransactionBlock } from "@mysten/sui.js/transactions";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: is this intentional?

Reply:
No, thanks! Happened from some weird linting behavior when creating the file.